chart/values.yaml

```diff
@@ -51,5 +71,23 @@ ssmsend:
   x509cert: aW5zZXJ0IGJhc2U2NC1lbmNvZGVkIHN0cmluZyBvZiB0aGUgWDUwOSBwdWJsaWMgY2VydCBmb3IgQVBFTCBwdWJsaXNoZXIK
   x509key: aW5zZXJ0IGJhc2U2NC1lbmNvZGVkIHN0cmluZyBvZiB0aGUgWDUwOSBwcml2YXRlIGtleSBmb3IgQVBFTCBwdWJsaXNoZXIK
 
+gratia:
+  resources:
+    limits:
+      cpu: "0.5"
+      memory: "500Mi"
+    requests:
+      cpu: "0.1"
+      memory: "50Mi"
+  # Location of gratia output container.
+  image_repository: "hub.opensciencegrid.org/osgpreview/kapel-gratia-output"
+  # Optionally overwrite container version. Default is chart appVersion.
```

Comment:
This comment is a bit misleading, the default here is actually "latest" but if a user manually sets `image_tag: ""` then .Chart.AppVersion would take effect in the cronjob template. 
Using latest is not recommended for production, so I would suggest a similar pattern to the ssmsend image_tag. Users can always set "latest" in their values if they want. Up to you though.

Reply:
Thanks for catching this, I've updated the `image_tag` value to `""` in the values.yaml